frontend/device/generic/device.lua

```diff
@@ -153,6 +153,9 @@ function Device:suspend() end
 -- Hardware specific method to resume the device
 function Device:resume() end
 
+-- Hardware specific method to power off the device
+function Device:power_off() end
```

Comment:
Yes it should, well spotted. I keep forgetting about the naming convention. Will fix.